.env.sample

```diff
@@ -4,4 +4,6 @@ SESSION_EXPIRY=
 SESSION_SECRET=
 OIDC_CLIENT_ID=
 AM_API_BASE_URL=
-OIDC_CLIENT_SCOPES=
\ No newline at end of file
+OIDC_CLIENT_SCOPES=
+GOV_PUBLISHING_API_BASE_URL_INT=
```

Comment:
you only need on value here as it is changed in the pipeline. I don't see this added to the pipeline either?